app/lib/meadow_web/router.ex

```diff
@@ -70,6 +70,13 @@ defmodule MeadowWeb.Router do
       before_send: {Middleware.AssumeRole, :update_user_role}
     )
 
+    forward("/graphiqlai", Absinthe.Plug.GraphiQL,
```

Comment:
I can't really explain why, but I'd be happier with this namespaced.
```suggestion
    forward("/graphiql/ai", Absinthe.Plug.GraphiQL,
```